tests/test_mbs_transformer.py

```diff
@@ -212,8 +212,11 @@ def test_no_dates_submitted(self):
         """
         Qid d12 is Yes and no Qid 11 or 12
         """
-        self.transformed_no_default_data["11"] is None
-        self.transformed_no_default_data["12"] is None
+        with self.assertRaises(KeyError):
+            self.transformed_no_default_data["11"] is None
+
+        with self.assertRaises(KeyError):
+            self.transformed_no_default_data["12"] is None
```

Comment:
Can the `is None` be dropped?

Reply:
It leaves a complaint in Codacy...

Reply:
Does it complain in codacy if the result is assigned to `_`?

Reply:
unfortunately it fails the lint then for some reason

Reply:
We should refine codacy for these cases

Reply:
So I'll take out the `is None`